src/store/index.js

```diff
@@ -33,8 +41,10 @@ export default function configureStore() {
     chats
   });
 
+  const persistedReducer = persistReducer(persistConfig, rootReducer)

```

Comment:
Don't forget about **semicolons** and use **eslint**, which Eugene send for you!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!